server/index.js

```diff
@@ -39,6 +39,29 @@ const upload = multer({
 app.use(staticMiddleware);
 app.use(jsonMiddleware);
 
+app.get('/api/download/:capsuleId', (req, res, next) => {
+  const { capsuleId } = req.params;
+  const sql = `
+    select content
+      from capsules
+     where "capsuleId" = $1
+  `;
+  const params = [capsuleId];
+  db.query(sql, params)
+    .then(result => {
+      const { content } = result.rows[0];
+      const splitUrl = content.split('/');
+      const key = splitUrl[splitUrl.length - 1];
```

Comment:
I recommend doing actual URL parsing to get the pathname instead, since it's possible that keys could include `/some-folder-type-thing`. And it's better if you can easily avoid having to update your code later when things like that happen.